modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -1,23 +1,19 @@
 // Module included in the following assemblies:
 //
-// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+// * scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc
 
-:_content-type: PROCEDURE
+:_content-type: CONCEPT
 [id="sno-du-enabling-workload-partitioning_{context}"]
-= Enabling workload partitioning
+= Workload partitioning
 
-A key feature to enable as part of a {sno} installation is workload partitioning. This limits the cores allowed to run platform services, maximizing the CPU core for application payloads. You must configure workload partitioning at cluster installation time.
+During {sno} cluster installation, you must enable workload partitioning. This limits the cores allowed to run platform services, maximizing the CPU core for application payloads.
 
 [NOTE]
 ====
-You can enable workload partitioning during the cluster installation process only. You cannot disable workload partitioning post-installation. However, you can reconfigure workload partitioning by updating the `cpu` value that you define in the `performanceprofile`, and in the MachineConfig CR in the following procedure. 
+Workload partitioning can only be enabled during cluster installation. You cannot disable workload partitioning post-installation. However, you can reconfigure workload partitioning by updating the `cpu` value that you define in the performance profile, and in the related `MachineConfig` CR.
```

Comment:
You could move the "custom resource" full description from line 16 to this one, because this line contains the first reference to the "CR" acronym in this module.